x/fees/types/params.go

```diff
@@ -0,0 +1,103 @@
+package types
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+// Parameter store key
+var (
+	DefaultFeesDenom                 = "adevi"
+	DefaultDeveloperPercentage       = uint64(50)
+	DefaultValidatorPercentage       = uint64(50)
+	ParamStoreKeyEnableFees          = []byte("EnableFees")
+	ParamStoreKeyFeesDenom           = []byte("FeesDenom")
+	ParamStoreKeyDeveloperPercentage = []byte("DeveloperPercentage")
+	ParamStoreKeyValidatorPercentage = []byte("ValidatorPercentage")
```

Comment:
It can make sense to keep it if we decide to burn part of the fees, like eip-1559.

Reply:
but then isn't the burned fee static (`burned = gasPrice * baseFee`)?

Reply:
For Ethereum, `gasPrice * baseFee` is the amount burned. But this is the amount that we now split between validators & devs. Because `gasPrice * priorityFee` should go entirely to the validators, as eip-1559 intends.
But what I propose, is to make it possible to split `gasPrice * baseFee` into 3 parts:
- x% to devs
- y % to validators
- x% burned
We can start with 50% devs, 50% validators and 0% burned, with these params modifiable by governance.
Reason: we may find out in time that it is good to burn part of this fee, from the same reasons eip-1559 does it.
Let me know what you think.

Reply:
Your proposal looks good to me. The only issue is that there's no prioritized mempool on Tendermint (until the next major release), so technically the priority fee doesn't serve any purpose

Reply:
We should also describe how the fee is split for non dynamic fee txs (i.e pre EIP1559, such us access list tx and legacy tx types)

Reply:
> The only thing to note is that there's no prioritized mempool on Tendermint (until the next major release), so technically the priority fee doesn't serve any purpose

i.e you can set it to 0 or 100k and it won't change the ordering